Terminal.Gui/Core/Application.cs

```diff
@@ -702,8 +702,13 @@ public static void Run (Toplevel view, Func<Exception, bool> errorHandler = null
 			var resume = true;
 			while (resume)
 			{
-				try
-				{
+#if DEBUG
```

Comment:
I think it's generally a bad idea to have DEBUG work differently than RELEASE.

Is this something that you needed for when YOU were debugging this, or something that is absolutely required anytime ANYONE is running DEBUG?

Reply:
For me is for anytime ANYONE is running DEBUG because is much faster to catch errors in the local where they occurred. Without this forces me to waste much more time to debug. This is the main enter and there no needs to catch here all the exceptions in DEBUG mode.